ci-operator/step-registry/redhat-chaos/cerberus/one-run/redhat-chaos-cerberus-one-run-commands.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+ls 
+
+export CERBERUS_KUBECONFIG=$KUBECONFIG
+export CERBERUS_WATCH_NAMESPACES="[^.*$]"
+export CERBERUS_IGNORE_PODS="[^installer*,^kube-burner*,^redhat-operators*,^certified-operators*,^collect-profiles*]"
+
+
+./cerberus/prow_run.sh
```

Comment:
@paigerube14 Can we add some documentation here on what exact check is being done in cerberus so users know what was expected in case there is a failure.